### ISSUE TITLE
New default when adding layers

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#4551]
Currently the default new layer on the plotting sub-command is a boxplot.  I have never wanted that as an added layer yet. If there is no layer, then I am fine with a boxplot.  But when (usually) there are existing layers then making the last layer the default is what we do elsewhere, i.e. keeping the last value.  And that would be good here.
So if we start with a line plot and add a layer, it sets initially also to line plot.  Etc.

**Comment by maxwellfundi** [comment]
This sounds reasonable to have for consistency as well. I had a similar question where I actually wanted the default layer to be the first in the alphabet, but I dont remember the reason why @dannyparsons  advised to have boxplot as the default layer. 

@dannyparsons  What do you think about this issue?

**Comment by dannyparsons** [comment]
Yes happy with the current layer being the default for the next layer. I would still keep boxplot as the default for the first layer as its common, the first alphabetically could be something very uncommon.

**Comment by rdstern** [comment]
I would be happy with boxplot as the default first layer.  I have never wanted it as the additional layer, but it would be a natural possible first layer when using the **Describe > General > Graphics** dialogue.

**Comment by maxwellfundi** [comment]
I think to some people boxplot is still a very uncommon plot and having it as a dropdown menu, then its intuitive that there more layers where the user can simply click and select the layer they want.
